src/Microsoft.AspNetCore.Razor.Design/project.json

```diff
@@ -12,7 +12,10 @@
       "taghelper",
       "aspnetcore",
       "taghelpers"
-    ]
+    ],
+    "files": {
+      "include": [ "buildCrossTargeting/*", "build/*" ]
```

Comment:
...for more context on that common infrastructure, see https://github.com/Microsoft/msbuild/pull/1131